### PR TITLE
Add transitions to cookie disclaimer

### PR DESCRIPTION
**Pull request by AndreasPB**
* Adds a fade + fly in/out effect to the cookie disclaimer

The one from here https://svelte.dev/tutorial/in-and-out

### CODE REVIEW

File: frontend/src/components/cookie_disclaimer.svelte [hunk 1]
```diff
@@ -11,7 +12,7 @@
 
 {#if loadDisclaimer}
   {#if !$cookieDisclaimer}
-    <div class="flex justify-center">
+    <div class="flex justify-center" transition:fly={{ y: 200, duration: 2000 }}>
```
Comment (Primdahl26) [inline frontend/src/components/cookie_disclaimer.svelte:15]: I feel like 2 seconds is a bit too slow, maybe 1 second or half a second?

**Review by Primdahl26 — CHANGES_REQUESTED** [review]
Also when you are scrolled down and start the animation, it will jump up to the first "bottom" of the screen. I think this can be fixed by moving the `CookieDisclaimer` component inside the main div.

File: frontend/src/components/cookie_disclaimer.svelte (hunk 1 above)
Comment (AndreasPB) [reply frontend/src/components/cookie_disclaimer.svelte]: I agree, changed it to 1, but changed in:transition to a fade instead. So fade in, fly out

**Review by Primdahl26 — APPROVED** [review]
Works nice after #135 was merged! :D

**Pull request merged**